hasura.planx.uk/migrations/1709208323190_run_sql_migration/up.sql

```diff
@@ -0,0 +1,54 @@
+DROP VIEW public.analytics_summary;
+
+CREATE
+OR REPLACE VIEW public.analytics_summary AS
+select
+	a.id as analytics_id,
+	al.id as analytics_log_id,
+	f.slug as service_slug,
+	t.slug as team_slug,
+	a.type as analytics_type,
+	a.created_at as analytics_created_at,
+	(user_agent -> 'os' ->> 'name') :: text AS operating_system,
+	(user_agent -> 'browser' ->> 'name') :: text AS browser,
+	(user_agent -> 'platform' ->> 'type') :: text AS platform,
+	referrer,
+	flow_direction,
+	metadata ->> 'change' as change_metadata,
+	metadata ->> 'back' as back_metadata,
+	metadata ->> 'selectedUrls' as selected_urls,
+	metadata ->> 'flag' as result_flag,
+	metadata -> 'flagSet' as result_flagset,
+	metadata -> 'displayText' ->> 'heading' as result_heading,
+	metadata -> 'result' ->> 'description' as result_description,
+	case
+		when has_clicked_help then metadata
+		else null
+	end as help_metadata,
```

Comment:
_non-blocking nit:_

I should have caught this before but I noticed that `help_metadata` is a little weird: 

```JSON      
{
  "node_title": "Tree consent",
  "node_id": "qwMn0D7gnF",
  "help_metadata": {
    "isAutoAnswered": false
  },
  "change_metadata": null,
  "back_metadata": null,
  "selected_urls": null,
  "has_clicked_help": true,
  "result_flag": null,
  "result_description": null,
  "result_flagset": null,
  "result_heading": null,
  "allow_list_answers": []
},
```

I think there's only ever help_metadata stored on Upload components with extra "info" where we'd actually want to see it:


<img width="1078" alt="Screenshot 2024-02-29 at 15 58 34" src="https://github.com/theopensystemslab/planx-new/assets/36415632/242be0c5-f2f3-4c4d-b51c-e4fb65b10d6d">

This does catch it: 

```JSON
{
  "node_title": "Upload and label",
  "node_id": "uZzVosWkeT",
  "help_metadata": {
    "test": "Roof plan",
    "isAutoAnswered": false
  },
  "change_metadata": null,
  "back_metadata": null,
  "selected_urls": null,
  "has_clicked_help": true,
  "result_flag": null,
  "result_description": null,
  "result_flagset": null,
  "result_heading": null,
  "allow_list_answers": []
},
```

Although I think it also erroneously catches the `"isAutoAnswered": false`. Not a massive deal but maybe the case could be updated to ignore `isAutoAnswered: false`? In most situations when a user hits "More Information" we don't add any metadata I believe.

Reply:
Yep - this is a bit weird tbh! I'll take another look in a follow up PR to see if I can come up with something cleaner/better 👍